json-dir-list/README.md

```diff
@@ -1,10 +1,10 @@
 # JSON Directory Listing Tool
-For the `unix_file` input format, supported by the [SSH
-plugin](../plugin-sftp), Shesmu will try to crawl the directory structure using
-a `find` command to produce JSON output. This works about as well as one could
-hope. If file names, user names, or group names have characters that are now
-allowed in JSON strings, it does not go well. This tool provides a more robust
-alternative that does the JSON encoding correctly. To install it:
+For the `unix_file` input format, supported by the SSH plugin, Shesmu will try
```

Comment:
Wasn't this already in plugin-sftp.md?

Reply:
I thought I deleted the bones of it and moved them here.

Reply:
Not what I'm seeing on github: 
![image](https://github.com/oicr-gsi/shesmu/assets/17747491/07c88786-c0c6-4177-8c63-ff56abe7bb31)